packages/common-utils/src/renderChartConfig.ts

```diff
@@ -958,7 +958,7 @@ async function translateMetricChartConfig(
         ...chartConfig,
         from: {
           ...from,
-          tableName: metricTables[MetricsDataType.Gauge],
+          tableName: metricTables[MetricsDataType.Sum],
```

Comment:
The `renderWhere` function isn't using the table when generating the substring part of the query. I updated it to match though because it looked funny in the code.